docs/ProblemStatementAndGoals/ProblemStatement.md

```diff
@@ -1,70 +1,94 @@
+
+Directly related to station access is a management system for contractors. This includes the ability to collect and distribute contract documentation, contact information, training, and other records.
+A key control management subsystem would be beneficial to view key distribution in real time, as this is currently managed in a separate application.
+A system to authenticate users at stations prior to access would improve visibility and protection.
+A digital key system should control access and entry to the station approved from a work order generated in the work order system.
+
+## 1.2 Inputs and Outputs
+
+### Inputs 
+* User login information for staff, internal contractors, and visitors.
+* Uploading of documents.
+* Signing of documents.
+* Completion of training.
+* Verification of arrivals and departures from the plant for contractors and internal staff.
+* Adding of new staff and contractors.
+
+### Outputs 
+* Station documentation and documentation authors.
+* Station maps and access protocol information.
+* Station forms.
+* Site contact information.
+* Information on projects.
+
+## 1.3 Stakeholders
+
+The stakeholder for this project is the City of Hamilton, Water Division. 
+Primary stakeholders with the City are the Facilities team, SCADA (Supervisory Control and Data Acquisition) team, and Corporate Security. 
+Depending on what is decided during the requirements gathering process, other stakeholders from the City may need to be included in the project, such as City IT.
+* City of Hamilton, Water Division. Primary stakeholder and client for the project.
+* Facilities Team. Subdivision of the primary stakeholder.
+* SCADA (Supervisory Control and Data Acquisition). Subdivision of the primary stakeholder.
+* Corporate Security. A stakeholder with an interest regarding Hamilton Water station security.
+
+## 1.4 Environment
+
+* Software: Windows 10 operating system, android, iOS.
+* Hardware: Laptop computers, tablets, smartphones.
+
+# 2 Goals
+| Goals | Rationale |
+| --- | --- |
+| | |
+| | |
+| | |
+# 3 Stretch Goals
+| Goals | Rationale |
+| --- | --- |
+| Demonstrate the advantage of a single centralized platform instead of multiple disconnected platforms.| Hamilton Water has applications which do not communicate between each other. Loss of information and working in “silos” is common as a result. Demonstrating the benefits of a platform which integrates the features of separate applications into one would be advantageous. |
+|Expand the platform to be a contract management system, capable of having contract management tools accessible to authenticated project managers. |This would enable a greater integration of station and contractor documentation directly into the projects that utilize this information. |
+| | |
+# 4 Challenge Level and Extras
+
+The challenge level for this project is a general challenge level. This designation was decided because we do not believe there will be a research element required.
```

Comment:
line lengths were supposed to be 80 chars iirc

Reply:
I tried indenting another level, but seemed to mess up the table formatting and displayed it a row below rather than in the column beside.